<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FramelessWindow</class>
 <widget class="QWidget" name="FramelessWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="windowFrame" native="true">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">#windowFrame{border:1px solid palette(highlight); background-color:palette(Window);}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="TitleWidget" name="windowTitlebar" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#windowTitlebar{border: 0px none palette(base); border-top-left-radius:5px; border-top-right-radius:5px; background-color:palette(shadow); height:20px;}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="spacer">
           <property name="minimumSize">
            <size>
             <width>4</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>4</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="IconWidget" name="icon" native="true">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">#icon {background-color:palette(shadow);}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TitleWidget" name="titleWidget" native="true"/>
         </item>
         <item>
          <widget class="CaptionButton" name="minimizeButton" native="true"/>
         </item>
         <item>
          <widget class="CaptionButton" name="restoreButton" native="true"/>
         </item>
         <item>
          <widget class="CaptionButton" name="maximizeButton" native="true"/>
         </item>
         <item>
          <widget class="CaptionButton" name="closeButton" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="windowContent" native="true">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#windowContent{
  border: 0px none palette(base);
  border-radius:0px 0px 5px 5px;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CaptionButton</class>
   <extends>QWidget</extends>
   <header>captionbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IconWidget</class>
   <extends>QWidget</extends>
   <header>iconwidget.h</header>
  </customwidget>
  <customwidget>
   <class>TitleWidget</class>
   <extends>QWidget</extends>
   <header>titlewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
